Revert upstream commit 4a5d98ee0. Details below:

From 4a5d98ee0a8650c651a781379787efc403a90585 Mon Sep 17 00:00:00 2001
From: Mariusz Felisiak <felisiak.mariusz@gmail.com>
Date: Tue, 31 May 2022 15:21:17 +0200
Subject: [PATCH] [3.2.x] Bumped minimum Sphinx version to 4.5.0.

Related Sphinx changes:
- https://github.com/sphinx-doc/sphinx/pull/8898
- https://github.com/sphinx-doc/sphinx/issues/8326

Backport of ebf25555bbed3e9112d4b726575d60b242daf48a from main.
---
 docs/conf.py | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 4f643e9dec06..fabe91cc3898 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -32,7 +32,7 @@
 # -- General configuration -----------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
+needs_sphinx = '1.6.0'
-needs_sphinx = "4.5.0"
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
@@ -70,8 +70,8 @@
 # The encoding of source files.
 # source_encoding = 'utf-8-sig'
 
+# The master toctree document.
+master_doc = 'contents'
-# The root toctree document.
-root_doc = "contents"
 
 # Disable auto-created table of contents entries for all domain objects (e.g.
 # functions, classes, attributes, etc.) in Sphinx 5.2+.
@@ -102,12 +102,12 @@ def django_release():
 django_next_version = '4.0'
 
 extlinks = {
+    'bpo': ('https://bugs.python.org/issue%s', 'bpo-'),
+    'commit': ('https://github.com/django/django/commit/%s', ''),
+    'cve': ('https://nvd.nist.gov/vuln/detail/CVE-%s', 'CVE-'),
-    "bpo": ("https://bugs.python.org/issue?@action=redirect&bpo=%s", "bpo-%s"),
-    "commit": ("https://github.com/django/django/commit/%s", "%s"),
-    "cve": ("https://nvd.nist.gov/vuln/detail/CVE-%s", "CVE-%s"),
     # A file or directory. GitHub redirects from blob to tree if needed.
+    'source': ('https://github.com/django/django/blob/main/%s', ''),
+    'ticket': ('https://code.djangoproject.com/ticket/%s', '#'),
-    "source": ("https://github.com/django/django/blob/main/%s", "%s"),
-    "ticket": ("https://code.djangoproject.com/ticket/%s", "#%s"),
 }
 
 # The language for content autogenerated by Sphinx. Refer to documentation
@@ -304,7 +304,7 @@ def django_release():
 # List of tuples (startdocname, targetname, title, author, dir_entry,
 # description, category, toctree_only)
 texinfo_documents = [(
+    master_doc, "django", "", "", "Django",
-    root_doc, "django", "", "", "Django",
     "Documentation of the Django framework", "Web development", False
 )]
 
